Makefile

```diff
@@ -48,13 +48,19 @@ lint:
 ## test: Running all *_test.go.
 test:
 	@echo "--> Running tests"
-	@go test -v ./...
+	@go test -v `go list ./... | grep -v node/tests`
 .PHONY: test
 
 ## test-race: Running all *_test.go with data race detector
 test-race:
 	@echo "--> Running tests with data race detector"
-	@go test -v -race ./...
+	@go test -v -race `go list ./... | grep -v node/tests`
+.PHONY: test-race
+
+## test-swamp: Running swamp tests located in node/tests
+test-swamp:
+	@echo "--> Running swamp tests"
+	@go test -v ./node/tests/swamp
```

Comment:
Taking into account your pov, I'll leave `test` as running everything with/without data race (will be around 1-2 mins or exeuction) 
In addition, I'm renaming the unit to be explicitly `test-unit` instead of previous `test` and `test-race`

Reply:
Awesome!